hydra_visualizer/src/layer_info.cpp

```diff
@@ -71,6 +72,22 @@ void declare_config(LayerConfig::Edges& config) {
   field(config.draw, "draw");
   field(config.scale, "scale");
   field(config.alpha, "alpha");
+  enum_field(config.color,
```

Comment:
Minor: you could define a config utilities enum type conversion, then no duplicates would be needed.